src/SequenceManager.cc

```diff
@@ -94,30 +96,36 @@ void store_sequence_parameters(const std::filesystem::path& lua_file, const Sequ
 } // anonymous namespace
 
 SequenceManager::SequenceManager(std::filesystem::path path)
-    : path_{ std::move(path) }
+    : path_{ std::move(path) }, gl_{path_}
 {
     if (path_.empty())
         throw Error("Base path name for sequences must not be empty");
 }
 
 Sequence
-SequenceManager::copy_sequence(UniqueId original_uid, const SequenceName& new_name) const
+SequenceManager::copy_sequence(UniqueId original_uid, const SequenceName& new_name)
 {
     const auto sequences = list_sequences();
     const UniqueId new_unique_id = create_unique_id(sequences);
 
     Sequence sequence = load_sequence(original_uid, sequences);
+    const std::string& old_name = sequence.get_name().string();
```

Comment:
We should not store a reference here because two lines later the name in the original object is changed, so `old_name` will actually change along with it. Instead, let's just store a normal string:
```cpp
const std::string old_name = sequence.get_name().string();
```

Reply:
Got removed in latest commit